meta-lmp-base/recipes-support/bluetooth-attach/bluetooth-attach/hciattach-custom.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ -z $1 || -z $2 || -z $3 ]]; then
+    echo Usage: $0 {tty} {speed} {proto}
+    exit
+fi
+
+if grep -qE "(fsl,imx8mm-evkb|fsl,imx8mp-evk)" /proc/device-tree/compatible &>/dev/null; then
```

Comment:
This script is bsp specific, be good to have it as an append in meta-lmp-bsp instead (we can change the service to only run when this script is available or create a generic script that can be replaced by one from meta-lmp-bsp.

Reply:
Done